core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -387,26 +404,14 @@ class WorkspaceService(protected val userInfo: UserInfo,
                   }
               }
             }
-
-            // Delete components of the workspace
-            _ <- dataAccess.submissionQuery.deleteFromDb(workspaceContext.workspaceIdAsUUID)
-            _ <- dataAccess.methodConfigurationQuery.deleteFromDb(workspaceContext.workspaceIdAsUUID)
-            _ <- dataAccess.entityQuery.deleteFromDb(workspaceContext.workspaceIdAsUUID)
-
-            // Delete the workspace
-            _ <- dataAccess.workspaceQuery.delete(workspaceName)
-
-            // Schedule bucket for deletion
-            _ <- dataAccess.pendingBucketDeletionQuery.save(PendingBucketDeletionRecord(workspaceContext.bucketName))
-
           } yield {
             workflowsToAbort
           }
         }
       }
     for {
 
-      workflowsToAbort <- deletionFuture recoverWith {
+      workflowsToAbort <- abortWorkflowFuture recoverWith {
         case t:Throwable => {
           logger.warn(s"Unexpected failure deleting workspace (while running `deletionFuture`) for workspace `${workspaceName}`", t)
```

Comment:
Update the log message, something like:

```suggestion
          logger.warn(s"Unexpected failure deleting workspace (while gathering workflows that need to be aborted) for workspace `${workspaceName}`", t)
```